packages/sampler/stories/qa/LabeledIcon.js

```diff
@@ -70,7 +71,7 @@ storiesOf('LabeledIcon', module)
 			const labelPosition = select('labelPosition', ['above', 'after', 'before', 'below', 'left', 'right'], Config);
 			return (
 				<LabeledIcon
-					icon={select('icon', ['', ...iconNames], Config, 'fullscreen')}
+					icon={select('icon', ['', ...sortedIconNames], Config, 'fullscreen')}
```

Comment:
This would also double up the empty string here.  Can either add it as above or use as you do in other places.

Reply:
I'd recommend doing the following for these, to reduce the amount of code duplication:
```
icon={select('icon', sortedIconNames, Config, 'fullscreen')}
```
Clearer to read, less duplication, no spreading, no extra spaces.

Reply:
Removed one of the 2 options and used 
```icon={select('icon', sortedIconNames, Config, 'fullscreen')}```